src/layer/LayersFactory.ts

```diff
@@ -332,6 +340,36 @@ export class LayersFactory {
     );
   }
 
+  private static async makeLayersWmsWmtMs(
```

Comment:
This could be merged with `makeLayersWms`as  it is duplicated except for the layer class.  I think one function with additional param layerClass would be better. 

Or perhaps even better option would be to  change  `fetchLayersFromGetCapabilitiesXml` a bit (extract parsing capabilities into own function) and use `fetchGetCapabilitiesXml` to get capabilities and determine with isWMSCapabilities which layer should be created. In this scenario we could also remove hardcoded url  above (services.terrascope.be/wms/v2)